src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -67,6 +67,211 @@ def test_WINNF_FT_S_SIQ_2(self):
     self.assertFalse(response['availableChannel'])
     self.assertEqual(response['response']['responseCode'], 0)
 
+  @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_3(self):
+    """Multiple CBSDs inside and outside the Exclusion Zone.
+
+    Response codes will vary depending on conditions.
+    This test assumes some distance criteria between points and zones.
+    If testdata records are changed, check again these criteria.
+    """
+
+    # Inject FSS station operating at range 3670 - 4200 MHz.
+    fss = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    fss['deploymentParam'][0]['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3670000000
+    fss['deploymentParam'][0]['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 4200000000
+    fss['deploymentParam'][0]['installationParam']['latitude'] = 38.4
+    fss['deploymentParam'][0]['installationParam']['longitude'] = -98.9
+    self._sas_admin.InjectFss({'record': fss})
+
+    # Inject GWBL station operating at range 3650 - 3700 MHz and associated
+    # GWPZ, within 150 km of FSS (approx. 100 km from FSS location set above).
+    gwpz = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+    self._sas_admin.InjectWisp(gwpz)
+
+    # Load PAL/PPA data, operating channel 3620 - 3630 MHz.
```

Comment:
Current TS-0061 seems to be 3600 - 3500 MHz for PAL.
"Admin Test Harness injects PPA Information into SAS UUT with an operating channel F within 3600-3650MHz frequency range."

Reply:
The PPA operating channel shall be 'within' 3600-3650 MHz, so I have chosen a 10 MHz channel (3620 to 3630 MHz) within this range. Should be ok I think.

Reply:
Ah yes, I agree. Thanks!